editoast/Cargo.toml

```diff
@@ -69,7 +70,7 @@ opentelemetry-datadog = { version = "0.10.0", features = ["reqwest-client"] }
 opentelemetry-otlp = "0.15.0"
 opentelemetry-semantic-conventions = "0.14.0"
 opentelemetry_sdk = { version = "0.22.1", features = ["rt-tokio", "trace"] }
-osm4routing = "0.6.1"
+osm4routing.workspace = true
```

Comment:
This crate doesn't belong in schema as it's OpenStreetMap related. We're only using it for a simple calculation in BoundingBox so I'd suggest to just inline the crate's function in editoast_schemas. Add a few lines of doc explaining what it does and why we're not using osm4routing (and credits Tristram for the implem 😁).

It's a shame that geos doesn't have this built in though.

Reply:
Done.